server/src/main/java/io/spine/server/entity/RecordBasedRepository.java

```diff
@@ -347,13 +320,13 @@ private E toEntity(EntityRecord record) {
      *
      * @return new instance of the transforming function
      */
-    protected final Function<Map.Entry<I, EntityRecord>, E> storageRecordToEntity() {
-        return new Function<Map.Entry<I, EntityRecord>, E>() {
+    protected final Function<EntityRecord, E> storageRecordToEntity() {
```

Comment:
Please change it in the following way:
 * Have the conversion function which is created on a repository initialization. 
 * Make it `Serializable`. It will be needed for integration with Beam.
 * Pass the instance of the function into `transform()`.
 * Call the function from the `toEntity()` method.